spec/order_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'order.rb'
+
+describe Order do
+  
+  it 'initializes with an empty order' do
+    expect(subject.order_placed).to eq [] 
+  end
+
+  describe "#add" do
+
+    it 'adds available dishes to order' do
+      # need to input a double here so test is unaffected by menu class
+      subject.add("sandwhich")
+      expect(subject.order_placed[0]).to eq("sandwhich")
```

Comment:
Could change order placed into a hash so you can store the item ordered as well as the quantity - so let's say they want three sandwiches so it would be something like `expect(subject.order_placed).to eq({sandwich => 3})` and `.add `would take two arguments